### ISSUE TITLE
[Unity] Detect if cardboard plugin is active and running or not

### ISSUE THREAD

**Issue by arjenveenhuizen** [googlevr/cardboard#282]
Platform: Android and iOS
SDK tested: Unity XR plugin v1.6.0
Problem statement: cannot determine if the cardboard XR plugin is active and loaded or not.

Background: I would like to determine on which HMD device my app is running, as it can run on many different HMDs.

So far, I wasn't able to find any API that can tell me that the Google Cardboard XR plugin is loaded and active or not.

For example `UnityEngine.XR.XRSettings.loadedDeviceName` only shows: `Display`  and 

```
            var xrDisplaySubsystems = new List<UnityEngine.XR.XRDisplaySubsystem>();
            SubsystemManager.GetInstances(xrDisplaySubsystems);
            foreach(UnityEngine.XR.XRDisplaySubsystem xrDisplaySubsystem in xrDisplaySubsystems) {
                UnityEngine.Debug.Log("Running? " + xrDisplaySubsystem.running);
            }
```

only shows `Running? true`.

But I cannot find a deterministic way to know that the cardboard xr plugin is active. Any idea?

**Comment by arjenveenhuizen** [comment]
A suggestion might be to set the name of the XRLoader to a more meaningful value?
ref. https://github.com/googlevr/cardboard-xr-plugin/blob/48e5eafae7d60bf65b5be9435a70b7234a474062/Runtime/XRLoader.cs#L84

**Comment by jballoffet** [comment]
Thanks for reaching out to us! A ticket for adding this feature has been added to our backlog and will be prioritized along with other tickets. We'll update this issue once we have some updates about it. In case you'd like to propose the required changes, please feel free to do so, just read CONTRIBUTING.md and follow the process. Patches are welcome.

**Comment by arjenveenhuizen** [comment]
I opened a PR that implements above's suggestion here: https://github.com/googlevr/cardboard-xr-plugin/pull/21

**Comment by jballoffet** [comment]
Thanks @arjenveenhuizen for contributing to Cardboard!

**Comment by xinyunh0929** [comment]
"This was fixed in v1.8.0."
